usecases/scheduler/src/lambda/ssv2_resultProcesser/resultProcesser.py

```diff
@@ -0,0 +1,263 @@
+import boto3 
+import json
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import os
+import openpyxl
+import logging
+
+## Initialization
+SnsEmailSubject = "[AWS] Service Screener Scheduler Report"
+SnsEmailBodyPrefix = "Here is your Service Screener report.\n\n"
+snsPrefix = os.environ['SSV2_SNSARN_PREFIX']
+
+
+successResp = {
+    'statusCode': 200,
+    'body': 'success'
+}
+
+currentResults = {}
+previousResults = {}
+accounts = {}
+
+SHEETS_TO_SKIP = ['Info', 'Appendix']
+
+def lambda_handler(event, context):
+    record = event['detail']
+    region = event['region']
+    targetBucket = record['bucket']['name']
+    targetObject = record['object']['key']
+    
+    s3 = boto3.client('s3', region_name=region)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

A Boto3 client is initialized from a Lambda handler. To speed up your Boto3 client initialization and minimize the operational cost of your Lambda function, we recommend that you create the client at the level of the module that contains the handler, and then reuse it between invocations.

[Learn more](https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html#function-code)

Similar issue at line number 33.